CharacterCreation/Class-Specific Documentation/Doctor/Doctor Hippocratic Proceedures.txt

```diff
@@ -0,0 +1,337 @@
+==============================
+Hippocratic Proceedures
+==============================
+
+==Patch Em Up ==
+Adrenaline Reward: 40 Points
+Contagion Reward: 1 Contagion 
+Range:          Touch
+Action:         2 Actions
+    
+    You use your biofoam appllicator to spray the smelly green foam into their
+wounds. The miracle of modern science rapidly reconnects arteries and veins and
+begins to mimic destroyed organs and muscle tissue. It hurts like hell, but it
+will give them a chance to keep it together long enough to get a more permanent
+solution back at base. 
+
+    You must have your Field Applicator drawn in order to use this ability. Your
+patient heals an ammount of health listed on the table below. As your character
+learns more medicine skill, the better they heal. Unless otherwise stated, this
+ability cannot heal a patient above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       9         |     20       | + 2d10 |
++-----------------+--------------+--------+
+|       19        |     30       | + 3d10 |
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
+
+==Defib ==
+Adrenaline Reward:  80 Points
+Contagion Reward: 2 Contagion
+Range: Touch
+Action: 2 Actions
+
+    You adjust the dials on your biofoam injector to compensate for loss of
+vital fluids and to administer the correct frequency and wattage of tachycardia
+interuptor shock. This is gonna hurt like hell, but they'll live to complain 
+about it at least. Who knows? Maybe they'll even say "Thank you!" this time?
+
+    You must have your Field Applicator drawn in order to use this ability. Only 
+usable on an organic patient that is bleeding out. The person is no longer 
+downed and has 10+2d10 Health Points, rather than having to stabilize the target 
+and then heal them as a separate action.
+
+== Pre-op traige ==
+Adrenaline Reward:  30 Points
+Contagion Reward: 1 Contagion
+Range: 10m
+Action: 1 Action
+
+    You scan the patient and dial your Field Applicator to match the patient's
+unique biochemestry. You also take the time to more closely inspect the patient's
```

Comment:
Dialing a device to match unique biochemistry sounds like more than one action to me. What if instead the ability requires that you have the target's chemistry already loaded as a preset on the applicator (and you could add such a preset during downtime or a rest maybe)?

Reply:
Add in some prose here that building a preset for an individual's biochem takes [some number of actions] and/or a DC [X] Medicine (INT) check, and is stored in the device's memory. Updating or validating the preset can be a free action if the doctor then needs to modify the baseline vitals and biochem.

Reply:
I agree with the strategy above. Could we break this into a future issue to un-block the huge feature change, and I'll get it this week?